helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -149,6 +149,33 @@ def _read_group_stage_ids(self, stages, domain, order):
         string="Automatic last update", default=datetime.now()
     )
 
+    ticket_change = fields.Datetime(
+        string="Next stage date", compute="_compute_next_stage"
+    )
+
+    @api.depends("stage_id")
```

Comment:
@cristina-domatix i think it can by done while depending only on `stage_id.auto_last_update` and `stage_id.auto_next_number` and not the full object